src/checks/key_without_value.rs

```diff
@@ -0,0 +1,51 @@
+use crate::checks::{Check, Warning};
+use crate::LineEntry;
+
+pub(crate) struct KeyWithoutValueChecker {
+    warning: Warning,
+}
+
+impl Default for KeyWithoutValueChecker {
+    fn default() -> Self {
+        Self {
+            warning: Warning::new("The {} key should be with a value or have an equal sign"),
+        }
+    }
+}
+
+impl Check for KeyWithoutValueChecker {
+    fn run(&self, line: &LineEntry) -> Option<Warning> {
+        if !line.raw_string.contains('=') {
+            Some(self.warning.clone())
```

Comment:
You should find a key and substitute it in a message to get the message like that: `The RAILES_ENV key should be with a value or have an equal sign`

Reply:
Fixed. :)